modules/exploits/multi/ssh/sshexec.rb

```diff
@@ -133,12 +172,11 @@ def do_login(ip, user, pass, port)
 
   def exploit
     do_login(datastore['RHOST'], datastore['USERNAME'], datastore['PASSWORD'], datastore['RPORT'])
-
     print_status("#{datastore['RHOST']}:#{datastore['RPORT']} - Sending stager...")
     if target['Platform'] == 'python'
       execute_command("python -c \"#{payload.encoded}\"")
     else
-      execute_cmdstager({:linemax => 500})
+      execute_cmdstager(linemax: 500)
```

Comment:
Should this be `execute_cmdstager({linemax: 500})`?  The brackets make it would be more clear that this is a map and not an named parameter.

Reply:
You can leave the braces off, which is a Ruby style-ism. However, you make a pretty good point about confusing it with a keyword arg!